apps/client/components/ui/Dashboard/DashboardContent.tsx

```diff
@@ -0,0 +1,57 @@
+import Targets from '@/components/table/Targets'
+import { Panel, PanelContent, PanelHeader } from '../Panel'
+import NominationBox from '@/components/Combobox'
+
+async function DashboardContent({
+  assignedReviews,
+  users,
+  cycleId,
+  loggedUserId,
+  report,
+}) {
+  return (
+    <div className="mt-10 grid grid-cols-3 max-w-screen h-[34rem] gap-10">
+      <div className="col-span-2 h-full grid grid-col-2 gap-10 max-w-screen">
+        <div className="border-2 grid-rows-1 border-black">
+          <NominationBox
+            users={users}
+            loggedUserId={loggedUserId}
+            report={report}
+            cycleId={cycleId}
+          ></NominationBox>
+          <Panel>
+            <PanelHeader>NOMINATION PHASE</PanelHeader>
+            <PanelContent>
+              {users.map((user) => (
```

Comment:
can you remove this? there shouldn't be any placeholder content when this gets merged